src/platform/renesas/CHIPDevicePlatformConfig.h

```diff
@@ -25,29 +25,14 @@
 #pragma once
 
 // -------------------- General Configuration --------------------
-#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE 0x00001300
+#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE 0x00002000
```

Comment:
I am trying to get it down to 0x1500 as I see that is the maximum usage 0x13F8 is the maximum usage right now.

Reply:
Settled on 0x1600